keras_cv/callbacks/pycoco_callback_test.py

```diff
@@ -65,9 +67,11 @@ def test_model_fit_rcnn(self):
         model = keras_cv.models.FasterRCNN(
             classes=10,
             bounding_box_format="xywh",
-            backbone=keras_cv.models.ResNet50V2(
-                include_top=False, include_rescaling=True, weights=None
-            ).as_backbone(),
+            backbone=keras_cv.models.ResNetV2Backbone.from_preset(
+                "resnet50_v2_imagenet",
+                load_weights=False,
+                include_rescaling=True,
```

Comment:
what's your intuition on `include_rescaling` at this time?  Are we still intending to include it in every from_preset call?

Reply:
Either we add `include_rescaling` as a positional arg in `from_preset` or we add a default in all the presets. From talking to the team I sense preference would be toward the former.

Reply:
Sgtm

Reply:
IMO we should default it to True and not require it anywhere.

Reply:
OK I will set it to `True` in all the presets and we can see how we feel.